BASE/src/Microsoft.ApplicationInsights/Metrics/Implementation/AutocollectedMetricsExtraction/RequestExtractor/RequestIdDimensionExtractor.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Metrics
+{
+    using System;
+    using Microsoft.ApplicationInsights.Channel;    
+
+    internal class RequestIdDimensionExtractor : IDimensionExtractor
```

Comment:
RequestId & DepednencyId are often associated with RequestTelemetry.Id and DependencyTelemetry.Id - should we consider slightly different name for those methods in extractors?